packages/stockflux-launcher/src/reducers/child-window/ChildWindow.js

```diff
@@ -0,0 +1,26 @@
+import STATE_TYPE from './State';
+import ACTION_TYPE from './Action';
```

Comment:
We already discussed this but not sure if you've gotten around to it yet so I thought I'd leave a comment just for your reference. Can we change these to `StateType` and `ActionType`?